content/intro-to-storybook/react-native/en/get-started.md

```diff
@@ -4,503 +4,187 @@ tocTitle: 'Get started'
-    fontSize: 14,
-    lineHeight: 20,
-    fontStyle: 'normal',
-  },
-  LoadingItem: {
-    alignItems: 'center',
-    backgroundColor: 'white',
-    flexWrap: 'nowrap',
-    flexDirection: 'row',
-    flex: 1,
-    height: 48,
-    justifyContent: 'space-around',
-    paddingLeft: 16,
-    width: '100%',
-  },
-  ListItems: {
-    flex: 1,
-    backgroundColor: 'white',
-    minHeight: 288,
-  },
-  WrapperMessage: {
-    position: 'absolute',
-    top: '40%',
-    right: 0,
-    bottom: 'auto',
-    left: 0,
-    width: 'auto',
-    height: 'auto',
-    textAlign: 'center',
-  },
-  PageListsShow: {
-    minHeight: '100vh',
-    backgroundColor: 'white',
-  },
-  PageListsShowhead: {
-    backgroundColor: '#d3edf4',
-    paddingTop: 24,
-    paddingBottom: 24,
-    paddingLeft: 20,
-    paddingRight: 20,
-    textAlign: 'center',
-  },
-  TitleMessage: {
-    fontSize: 16,
-    lineHeight: 24,
-    fontFamily: 'NunitoSans',
-    fontWeight: '800',
-    color: '#555',
-  },
-  SubtitleMessage: {
-    fontSize: 14,
-    lineHeight: 20,
-    color: '#666',
-    fontFamily: 'NunitoSans',
-  },
-  titlepage: {
-    fontSize: 20,
-    lineHeight: 24,
-  },
-  TitleWrapper: {
-    fontFamily: 'NunitoSans',
-    fontWeight: '800',
-    color: '#1c3f53',
-    maxWidth: '100%',
-  },
-});
+```shell
+npx -p @storybook/cli sb init --type react_native
 ```
 
-</details>
+This should generate a folder in your project called .storybook, this is where the config for storybook goes.
 
-![Taskbox UI](/intro-to-storybook/ss-browserchrome-taskbox-learnstorybook.png)
+Now add this to your App.js file and comment out the App function for now.
 
-<div class="aside">
-If you want to modify the styling, the source LESS files are provided in the GitHub repo. And adjust accordingly for React Native styling.
-</div>
+```js:title=App.js
+export { default } from './.storybook';
+```
 
-## Add assets
+If you run the app again with `yarn ios` you should now see this:
 
-To match the intended design, you'll need to download both the font and icon directories and place them inside the `assets` folder.
+![image showing button in the storybook ui](/intro-to-storybook/react-native-hello-world.png)
 
-<div class="aside">
-<p>We’ve used <code>svn</code> (Subversion) to easily download a folder of files from GitHub. If you don’t have subversion installed or want to just do it manually, you can grab the icons folder directly <a href="https://github.com/chromaui/learnstorybook-code/tree/master/src/assets">here</a> and the font <a href="https://github.com/google/fonts/tree/master/ofl/nunitosans">here</a>.</p></div>
+If what you see on your device matches this image then you've successfully setup storybook.
 
-```shell:clipboard=false
-svn export https://github.com/chromaui/learnstorybook-code/branches/master/src/assets/icon assets/icon
-svn export https://github.com/google/fonts/trunk/ofl/nunitosans assets/font
-```
+## Swap between storybook and app development
```

Comment:
I'm not sure cross-environment is the right wording here because that makes me think of ios/android or prod/staging. Is there another way we could word the heading? Maybe something like `How it works` or along those lines.

Reply:
Also I like the git section, however at this point we haven't made any changes so wouldn't there be nothing to commit?

Reply:
@jonniebigodes I've made most of these changes however I used `How it works`, let me know what you think

Reply:
Marking as solved. Agree with the reasoning and choice of words for this case. Thanks 🙏